src/controller/users.js

```diff
@@ -1,26 +1,42 @@
 const router = require("express").Router(),
-	updateUser = require("../actions/update-user"),
-	findUser = require("../actions/find-user");
+	auth = require("../middlewares/bearer"),
+	updateUser = require("../actions/users/update-user"),
+	findUser = require("../actions/users/find-user"),
+	notLoggedIn = {
```

Comment:
Since we might be repeating this a lot, consider putting this JSON block into a seperate file, and `require` it. This way you can more easily guarantee the 403 message is the same everywhere, and update them all in a single file edit.